### PR TITLE
Karam dosa

### PR DESCRIPTION
**Pull request by anil444999**

### CODE REVIEW

**Comment by anil444999** [comment]
why you add oil to the karam dosa? already oil is there for plain dosa? why you added specifically

**Comment by anilkumar333999** [comment]
hi to add

**Comment by anilkumar333999** [comment]
why you add oil to the karam dosa? already oil is there for plain dosa? why you added specifically

**Comment by anil444999** [comment]
> why you add oil to the karam dosa? already oil is there for plain dosa? why you added specifically

extra oil is good for karam dosa

File: menu.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-dosa
\ No newline at end of file
+dosa
+karam
+oil
```
Comment (anilkumar333999) [inline menu.txt:3]: why oil is added at the end?

Comment (anil444999) [inline menu.txt:3]: I added at the end, so it may be tasty

**Review by anilkumar333999 — APPROVED** [review]
everything is good, we can go ahead

**Pull request merged**